### DISCUSSION TITLE
linux mint 20 cinnamon 下任务栏无图标

### DISCUSSION THREAD

**Discussion by catcherxuefeng** [qier222/YesPlayMusic#1170] [Q&A] [answered]
# 尽量每个 issue 只提一个 bug 或新功能

### 提新 issue 前请确认 👉

- 没人提过这个 issue（[这里看所有 issue](https://github.com/qier222/YesPlayMusic/issues)）
- 项目的 Todo 里没有与你 issue 相关的内容（[这里看 Todo](https://github.com/qier222/YesPlayMusic/projects/1)）

### 反馈 bug 需要的信息

- 客户端 YesPlayMusic-0.4.3_c895133afbaf0b5ea3194a6ddc425e8a.AppImage
- Linux Mint 20.2 Uma
[![TbaHln.png](https://s4.ax1x.com/2022/01/03/TbaHln.png)](https://imgtu.com/i/TbaHln)

**Comment by memorydream** [comment]
这样子看不出来什么，是只有yesplaymusic会这样吗？其他的electron程序呢？

**Comment by catcherxuefeng** [comment, marked as answer]
> 这样子看不出来什么，是只有yesplaymusic会这样吗？其他的electron程序呢？

尝试了下其他的electron AppImage 程序，一样的问题，可能是发行版的问题
